native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -152,3 +142,29 @@ extension ViewController{
         return String(hour) + time.suffix(time.count - 2)
     }
 }
+
+extension ViewController{
+
+    func setUpSteppers() {
+        adultStepper.autorepeat = true
+        adultStepper.maximumValue = maxNumberTickets
```

Comment:
Is the max number of tickets for both adults and children. Or a combined total? i.e. is it does adult + children <= max number of tickets  or ((adult <= max) and (children <= max)).